checks/caesar.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
```

Comment:
This can't be put into a parent class because it is what allows us to access the parent class. 

Once check50 is installed as a package locally, this won't be necessary